locations/spiders/yuppiechef_za.py

```
@@ -2,19 +2,21 @@
 
 
-class YuppiechefZASpider(PlaywrightSpider):
+class YuppiechefZASpider(CamoufoxSpider):
     name = "yuppiechef_za"
     item_attributes = {"brand": "Yuppiechef", "brand_wikidata": "Q24234053"}
     start_urls = ["https://www.yuppiechef.com/store-directory.htm"]
-    custom_settings = DEFAULT_PLAYWRIGHT_SETTINGS_WITH_EXT_JS
+    custom_settings = DEFAULT_CAMOUFOX_SETTINGS_FOR_CLOUDFLARE_TURNSTILE
     no_refs = True
+    captcha_selector_indicating_success = '//article[contains(@class, "store-loc-card")]'
+    handle_httpstatus_list = [403]
```

Ruff (0.16.3)

[warning] Mutable default value for class attribute

(RUF012)